wiki/en/en-Installation-for-Windows.md

```diff
@@ -10,7 +10,8 @@ permalink: "/wiki/Installation-for-Windows"
 # Installation for Windows
 Make sure you read the [Getting Started](Getting-Started) page.
 1. **Download and install an ASIO Driver**. It is recommended to use a sound card/interface with a native ASIO driver. If you don't have an external sound card, you probably need to install [this free ASIO driver (ASIO4ALL)](https://www.asio4all.org){: target="_blank" rel="noopener noreferrer"} before installing Jamulus.
```

Comment:
Can this also note that "It's well worth installing the ASIO4ALL Off-line Settings tool when installing ASIO4ALL", as well?  It lets people set up before running Jamulus - i.e. not having Jamulus make wrong decisions about what devices are available.  Indeed, I'd even say this is the point to say "If you're installing ASIO4ALL and haven't used it before - go read **this**" and link to something that will explain what's going on...  Once we have it.

Reply:
> If you're installing ASIO4ALL and haven't used it before - go read this

Yes. That's a good suggestion. Can you open a PR to changes with this proposal?

Reply:
> You can't setup the A4A in/outputs with off line settings since A4A saves configuration on per app basis.

Oh - OK, no point then, agreed.  I wonder why it exists?  (Maybe that's why it's so hard to install....)